File: solutions/_1768_merge_strings_alternately/solution.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        result = ""
        length = max(len(word1), len(word2))
        for i in range(length):
            if i < len(word1):
                result += word1[i]
            if i < len(word2):
                result += word2[i]
        return result


# https://leetcode.com/problems/merge-strings-alternately/

File: solutions/_1768_merge_strings_alternately/solution.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function (word1, word2) {
  let result = "";
  const length = Math.max(word1.length, word2.length);
  for (let i = 0; i < length; i++) {
    if (i < word1.length) {
      result += word1[i];
    }
    if (i < word2.length) {
      result += word2[i];
    }
  }
  return result;
};

module.exports = { mergeAlternately };

/*
https://leetcode.com/problems/merge-strings-alternately/
*/
